src/main/java/PetService.java

```diff
@@ -0,0 +1,44 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PetService {
+    private PetSerializer serializer = new PetSerializer();
+    private File existingPets = new File("src/main/resources/Pets.json");
```

Comment:
Taking data from the file requires more complex solution. E.g. handle NotSuchFile and other exceptions 
Also it needs to open and close resources